Assets/HoloToolkit/Input/Tests/Scripts/SimpleGridGenerator.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class SimpleGridGenerator : MonoBehaviour
```

Comment:
Issue #71 is not being able to send messages to dynamically created objects, and I was asked to create test scenes to illustrate how this would work in a real scenario. I understand the rationale to keep HoloToolkit as simple as possible, but removing dynamic objects would defeat the purpose of the test case and obscure the intent of the proposal.

I think it would only help developers top have as many examples as possible.

If the desire is to keep HoloToolkit simple, I would suggest adding a separate Examples/Test folder under the root HoloToolkit folder instead of keeping Test folders alongside each module. Then, tests and examples can be excluded when importing a clean version of HoloToolkit into a production project.